manuscripts2/Readme.md

```diff
@@ -0,0 +1,230 @@
+		- `get_min`: get the minimum value from all the values in the said field (field should be numeric)
+		- `get_max`: get the maximum value from all the values in the said field (field should be numeric)
+		- `get_extended_stats`: get the extended statistics (variance, standard deviation, and so on) for the values in the said field (field should be numeric)
+
+	- Non Numeric:
+		- `get_terms`: get term aggregation for the said field
+**NOTE:** the `get_aggs()` function returns ony the numeric values, so in the case of `get_terms` aggregation, it will return the total count of the aggregation. It is better to use the `fetch_aggregation_results` function to get the individual terms instead.
+	
+- There is also an `add_custom_aggregation` filter which takes in an `elasticsearch_dsl Aggregation` object as it's input and adds it to the `aggregations` dict of the object (PullRequests, Issues, EQCC).
+
+---
+
+##### EXAMPLE 3: Get all the closed issues by authors.
+
+```python
+from new_functions import Index
+from derived_classes import Issues
+github_index = Index(index="<github_index_name>", url="<elsaticsearch address>")
+issues = Issues(github_index)
+issues.is_closed() # this add the filter {"state":"closed"} thus we look at only closed issues
+issues.get_cardinality("id_in_repo").by_authors("author_name")
+issues.fetch_aggregation_results()
+```
+
+**Steps:**
+
+- Create the index object
+- Create an Issues object with `github_index` as one of it's paremeters
+- Apply the `is_closed` filter to look at closed issues
+- Apply the aggregation to get cardinality (number of issues). Apply the `by_authors` aggregation which becomes the parent	aggregation for the `cardinality` aggregation. This step will actually pop the last added aggregation from theaggregation list (here the 'cardinality' agg) and add it as a child agg for `terms` aggregation where field is the`author_name`.
+- Call the `fetch_aggregation_results` function to get the number of closed issues by authors.
+
+**NOTE:**`fetch_aggregation_results` loops through all the aggregations that have been added to the Object (here: `issues`) and adds them to the Search object in the sequence in which they were added. Then it queries elasticsearch using the `Search().execute()` method and returns a dict of the values that it gets from elasticsearch.
+This will return a response from elasticsearch in the form of a dictionary having aggregations as one of the keys. The value for that(a dict itself) will have '0' as a key with the value containing the total number of unique authors in the repo 
+who created an issue/pr.
+
+**Points to Note:**
+
+- Aggregations similar to `by_author` are:
+	- `by_organizations`: It is similar to `by_authors` and is used to seggregate based on the organizations that the users belong to
+	- `by_period`: This creates a `date_histogram` aggregation.
+
+---
+
+##### EXAMPLE 4: Moar chainability
```

Comment:
More instead of moar?

Reply:
I was being creative :P https://en.wiktionary.org/wiki/moar

Reply:
;-) I didn't know about this "moar". Thanks for teaching me!!